pkg/provider/managedk8s/ruleset/disak8sstig/rules/242400.go

```diff
@@ -202,9 +203,9 @@ func (r *Rule242400) checkKubeProxy(
 	for _, pod := range pods {
 		podTarget := rule.NewTarget("name", pod.Name, "namespace", pod.Namespace, "kind", "pod")
 
-		rawKubeProxyCommand, err := kubeutils.GetContainerCommand(pod, "kube-proxy")
+		rawKubeProxyCommand, err := kubeutils.GetContainerCommand(pod, kubeProxyContainerNames...)
 		if err != nil {
-			checkResults = append(checkResults, rule.ErroredCheckResult(err.Error(), podTarget))
+			checkResults = append(checkResults, rule.ErroredCheckResult("Failed to find any containers appropriate for evaluation", podTarget.With("details", err.Error())))
```

Comment:
```suggestion
			checkResults = append(checkResults, rule.ErroredCheckResult(err.Error(), podTarget))
```
The error from `kubeutils.GetContainerCommand`. Note: we normally do not add error message in target `details`